### PR TITLE
Add grisp pack command

### PR DESCRIPTION
**Pull request by sylane**
Required by https://github.com/grisp/rebar3_grisp/pull/85

### CODE REVIEW

**Comment by sylane** [comment]
The dependency will be fixed when testing and review is done, and I can publish grisp_update_packager

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ and this project adheres to
   actions will **NOT** be called if the actions do not raise any exception.
   [#25](https://github.com/grisp/grisp_tools/pull/25)
 - Add a firmware command to generate GRiSP 2 binary firmwares: [#26](https://github.com/grisp/grisp_tools/pull/26)
+- Add a pack command to generate a GRiSP 2 software update package: [#27](https://github.com/grisp/grisp_tools/pull/27)
```
Comment (ziopio) [inline CHANGELOG.md]: wrong link should be this PR 28

File: src/grisp_tools.hrl [hunk 2]
```diff
@@ -0,0 +1,22 @@
+-ifndef(GRISP_TOOLS_HRL).
+-define(GRISP_TOOLS_HRL, true).
+
+-define(MiB, * 1024 * 1024).
+-define(DOCKER_TOOLCHAIN_ROOT, "/grisp2-rtems-toolchain/rtems/5").
+-define(GRISP2_BOOTLOADER_FILENAMES, [
+    "foobar.img",
```
Comment (ziopio) [inline src/grisp_tools.hrl]: foobar ? what is this?

File: src/grisp_tools.hrl [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "barebox-phytec-phycore-imx6ull-emmc-512mb.img",
+    "barebox-phytec-phycore-imx6ul-emmc-512mb.img"
```
Comment (ziopio) [inline src/grisp_tools.hrl:8]: do we need both of these in for GRiSP2 ?

**Review by ziopio — CHANGES_REQUESTED** [review]
REmember to cross link this PR with its rebar3_grisp sibling

**Review by GwendalLaurent — APPROVED** [review]

File: src/grisp_tools.hrl (hunk 3 above)
Comment (sylane) [reply src/grisp_tools.hrl:8]: Not sure if we need both, but the toolchain that is published on the wiki still uses the old bootloader name

File: src/grisp_tools.hrl (hunk 2 above)
Comment (sylane) [reply src/grisp_tools.hrl]: Some test boot loader that I forgot to remove, it is fixed.

**Comment by sylane** [comment]
> REmember to cross link this PR with its rebar3_grisp sibling

@ziopio Do you mean here in the PR description or somewhere else ?

**Comment by ziopio** [comment]
@sylane cross link the PRs in their description so it is easy to navigate between them and find them in future

**Review by ziopio — APPROVED** [review]

**Pull request merged**